profile/README.md

```diff
@@ -0,0 +1,54 @@
+# We are NewAperio
+
+![hero images](https://user-images.githubusercontent.com/1529366/173092313-2938e23d-8e4b-4d93-87f8-c827d38c4002.png)
+
+### What we do:
+
+We’re an agile, fully distributed team of developers and designers who sweat the details and take pride in our craft. We design and build web and mobile apps for clients in:
+- Healthcare 🩺 🏥 💊
+- Fitness 👟🥦🏋🏽
+- Education 🏫 ✏️🎓
+- Civic engagement 🗳️🪧📣
+- Developer tools 🕹️🛠️🤖
+
+In addition to our work in these core industries, we have completed numerous projects in other domains such as an app for conducting remote property inspections, an app that makes it easier to manage receipts and track expenses, and an app that turns codebases into visual maps.
+
+We also provide staff augmentation, so if your team is looking for developers or designers with particular technical expertise, or for expanded capacity during a busy period, let’s connect.
+
+### We are experts in:
+- Elixir
+- Phoenix
+- JavaScript
+- TypeScript
+- React
+- React Native
+- GraphQL
+- Postgres
+- Product Design Sprints
+- UI Design 
+- UX Design
+- Design Systems
+
+### Our history and values:
+
+Logan Leger founded NewAperio in 2010 while a student at Louisiana State University. The company has grown steadily since to include multiple developers and designers prototyping, designing, and building apps for clients nationwide. 
```

Comment:
```suggestion
[Logan Leger](https://github.com/lleger) founded NewAperio in 2010 while a student at Louisiana State University. The company has grown steadily since to include multiple developers and designers prototyping, designing, and building apps for clients nationwide. 
```

Wondering if linking here is a good idea. For those viewing for jobs in the future might increase cred.